shenyu-admin/src/main/resources/mappers/rule-sqlmap.xml

```diff
@@ -81,63 +81,62 @@
     </select>
 
     <select id="findBySelectorId" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
+        SELECT
         <include refid="Base_Column_List"/>
-        from rule where selector_id = #{selectorId,jdbcType=VARCHAR}
+        FROM rule WHERE selector_id = #{selectorId,jdbcType=VARCHAR}
     </select>
 
     <select id="findBySelectorIds" parameterType="java.util.List" resultMap="BaseResultMap">
-        select
+        SELECT
         <include refid="Base_Column_List"/>
-        from rule where selector_id in
+        FROM rule WHERE selector_id IN
         <foreach item="selectId" collection="list" open="(" separator="," close=")">
             #{selectId, jdbcType=VARCHAR}
         </foreach>
     </select>
 
     <select id="countByQuery" parameterType="org.apache.shenyu.admin.model.query.RuleQuery"
             resultType="java.lang.Integer">
-        select count(1)
-        from rule
+        SELECT count(1)
+        FROM rule
         <where>
             <if test="selectorId != null">
                 selector_id = #{selectorId,jdbcType=VARCHAR}
             </if>
             <if test="name != null and name != ''">
-                and name = #{name,jdbcType=VARCHAR}
+                AND name = #{name,jdbcType=VARCHAR}
             </if>
             <if test="filterIds != null">
-                and id in
+                AND id IN
                 <foreach item="filterId" collection="filterIds" open="(" separator="," close=")">
                     #{filterId, jdbcType=VARCHAR}
                 </foreach>
             </if>
         </where>
-
     </select>
 
     <select id="selectAll" resultType="org.apache.shenyu.admin.model.entity.RuleDO" resultMap="BaseResultMap">
-        select
+        SELECT
         <include refid="Base_Column_List"/>
-        from rule
+        FROM rule
     </select>
 
     <select id="existed" resultType="java.lang.Boolean">
-        select true from rule where id = #{id} limit 1
+        SELECT true FROM rule WHERE id = #{id} limit 1
     </select>
 
     <insert id="insert" parameterType="org.apache.shenyu.admin.model.entity.RuleDO">
-        insert into rule (id, date_created, date_updated,
+        INSERT INTO rule (id, date_created, date_updated,
```

Comment:
The keywords are right-aligned and the fields are left-aligned and displayed in multiple lines.